extensions/theme-defaults/themes/dark_modern.json

```diff
@@ -113,7 +113,7 @@
 		"textCodeBlock.background": "#2B2B2B",
 		"textLink.activeForeground": "#4daafc",
 		"textLink.foreground": "#4daafc",
-		"textPreformat.foreground": "#D0D0D0",
+		"textPreformat.foreground": "#D7BA7D",
```

Comment:
This is required for `code` blocks in Markdown previews in Dark Modern to render in a distinct color